src/frequenz/channels/_receiver.py

```diff
@@ -373,9 +400,7 @@ def consume(self) -> MappedMessageT_co:  # noqa: DOC502
             ReceiverStoppedError: If the receiver stopped producing messages.
             ReceiverError: If there is a problem with the receiver.
         """
-        return self._mapping_function(
-            self._receiver.consume()
-        )  # pylint: disable=protected-access
+        return self._mapping_function(self._receiver.consume())
```

Comment:
Optional: This is unrelated to the commit, could be moved to another commit.

Reply:
done.